charts/monitoring-config/rules/email_alert_api_alerts.yaml

```
@@ -0,0 +1,24 @@
+groups:
+  - name: EmailAlertAPIAlerts
+    rules:
+      - alert: TravelAdviceAlert
+        expr: >
+          email_alert_api_current_travel_advice_alerts -
+            email_alert_api_delivered_travel_advice_alerts > 0
+        labels:
+          severity: page
+        annotations:
+          summary: No delivered emails for Travel Advice update created >1 hour ago
```

maybe slightly clearer like this? what you reckon?
```suggestion
          summary: No delivery reports for Travel Advice update created >1 hour ago
```

---

Yeah, that makes sense (it's also more truthful, since even with our fallback it doesn't necessarily mean no emails were delivered).